layouts/partials/footer-scripts.html

```diff
@@ -7,6 +7,13 @@
 {{- $sourcemap := cond $isProd "external" "inline" -}}
 {{- $inject := slice "scripts/injects/inject-jquery.js" -}}
 
+{{- if .HasShortcode "jqmath-vanilla" -}}
+  {{- $opts := dict "targetPath" "static/jqmath.js" "defines" $defines "inject" $inject "sourceMap" $sourcemap "minify" $isProd -}}
```

Comment:
do we need to inject jquery here?

Reply:
You are right, I've removed the jquery inject and in this case it still works and reduces the js size 👍